WordPress/gutenberg
# FontSizeControl (Next): Fix SelectDropdown voiceover focus issues

This update fixes the voiceover focus interactions for the  (next) `FontSizeControl` component (recently merged https://github.com/WordPress/gutenberg/pull/27594)

The solution was to update to the latest `@wp-g2` version (v0.0.143):
https://github.com/ItsJonQ/g2/releases

These changes include updates to `z-index` and interaction fixes to the underlying `SelectDropdown` component (https://github.com/ItsJonQ/g2/pull/239).

Potentially resolves https://github.com/WordPress/gutenberg/issues/28411

## How has this been tested?

Tested locally on Gutenberg with Chrome + Firefox + Voiceover (Mac OS)
Also tested in the `@wp-g2` [pull request](https://github.com/ItsJonQ/g2/pull/239) in a similar manner.

To test:

* run `npm run dev`
* Click on a text-based block (e.g. Paragraph)
* Ensure the font size picker interactions are working as intended

## Checklist:
- [n/a] My code follows the WordPress code style. 
- [n/a] My code has proper inline documentation. 
- [n/a] I've included developer documentation if appropriate. 
- [n/a] I've updated all React Native files affected by any refactorings/renamings in this PR.
